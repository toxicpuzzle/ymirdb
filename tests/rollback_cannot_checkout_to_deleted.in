SET a 1
SNAPSHOT  
SET b 2
SNAPSHOT
SET c 3
SNAPSHOT
LIST SNAPSHOTS
LIST ENTRIES
ROLLBACK 1
DEL a
LIST ENTRIES
CHECKOUT 1
LIST ENTRIES
ROLLBACK 1
LIST ENTRIES
CHECKOUT 2
CHECKOUT 0
LIST ENTRIES
LIST SNAPSHOTS
DROP 1
LIST ENTRIES
CHECKOUT 1
bye