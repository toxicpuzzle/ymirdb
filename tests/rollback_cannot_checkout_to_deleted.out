> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> 3
2
1

> c [3]
b [2]
a [1]

> ok

> ok

> no entries

> ok

> a [1]

> ok

> a [1]

> no such snapshot

> no such snapshot

> a [1]

> 1

> ok

> a [1]

> no such snapshot

> bye
